docs/deployment/setup.md

```diff
@@ -0,0 +1,117 @@
+# KV-Cache Manager Setup Guide
+
+This guide provides a complete walkthrough for setting up and testing the example llm-d-kv-cache-manager system. You will deploy a vLLM with LMCache and Redis, then run an example application that demonstrates KV cache indexing capabilities.
+
+By following this guide, you will:
+
+1. **Deploy the Infrastructure**: Use Helm to set up a Kubernetes cluster running:
+   - vLLM servers (default: 4 replicas) serving Llama 3.1 8B Instruct model,
+   - Redis server,
+   - LMCache,
+2. **Test with Example Application**: Run a Go application that:
+   - Connects to your deployed vLLM and Redis infrastructure,
+   - Demonstrates KV cache indexing by processing a sample prompt,
+   - Shows how to retrieve pod scores, 
+
+The end result is a working distributed LLM system with intelligent KV cache management that can route requests to pods with relevant cached computations.
+
+## vLLM Deployment
+
+The llm-d-kv-cache-manager repository includes a Helm chart for deploying vLLM with LMCache and Redis. This section describes how to use this Helm chart for a complete deployment.
```

Comment:
s/vLLM with LMCache and Redis/vLLM with CPU offloading (LMCache) and KV-events indexing (Redis)/.